views/results/_card.html

```diff
@@ -1,12 +1,51 @@
-Keyword: {{ .result.Keyword }} <br>
-Status: {{ .result.Status }} <br>
-AdLinks: <br>
-{{ range $adLink := .result.AdLinks }}
-    {{ $adLink.Position }} {{ $adLink.Type }} {{ $adLink.Link }} <br>
-{{ end }}
-Non-AdLinks: <br>
-{{ range $link := .result.Links }}
-    {{ $link.Link }} <br>
-{{ end }}
-Page Cache: <br>
-<a href="/results/{{.result.Id}}/cache" >Page Cache</a>
+<div class="card">
+    <div class="card-header">
+        <h3 class="card-title">
```

Comment:
Not sure if this should be `card-header__title`?

Reply:
@gutakk I didn't use `card-header__title` because on some page I want to reuse the same `card-title` style but the component is not inside the `card-header` for example on the [upload card](https://github.com/carryall/go-google-scraper-challenge/pull/73/files/e56b0d011b16ec6a1eb32dff61804c379e786dfe#diff-8732eeba9b2bbca1342adf882841f1e5ec7f83254cea8746e0f07db7cdbf5d0e) there is no `card-header` and everything is inside the `card-body` instead